### PR TITLE
Parameter values

### PR DESCRIPTION
**Pull request by Shushman**
Provides the ability to query specific parameter values through a `GetParameterValue` command.

### CODE REVIEW

File: src/OMPLPlanner.cpp [hunk 1]
```diff
@@ -419,6 +425,56 @@ bool OMPLPlanner::GetParametersCommand(std::ostream &sout, std::istream &sin) co
...
+
+    bool in_map = false;
+    //Iterate through parameter map and return when it matches required param
+    ParamMap::const_iterator it;
+    for (it = param_map.begin(); it != param_map.end(); ++it) {
+        if (it->first == inp_arg){
+            in_map = true;
+            sout<<it->first<<" "<<it->second->getValue();
+            break;
+        }
```
Comment (mkoval) [inline src/OMPLPlanner.cpp]: This is doing a linear search over parameters. Can you replace this with a call to `getParam`()`.

**Comment by mkoval** [comment]
@Shushman I made one comment. I'm ready to merge this once you make that change.

**Comment by mkoval** [comment]
@Shushman I'd like to merge this. Could you make the small change I suggested above?

**Comment by Shushman** [comment]
I've addressed the change. It works with `getParam()` now.

**Comment by cdellin** [comment]
This merged PR includes code in `OMPLPlanner::IsInOrCollision` which appends the location of every collision check to a file ... I assume that was an oversight?

**Comment by Shushman** [comment]
Damn. Must have overlooked that amidst everything. I'll change it ASAP.

**Comment by psigen** [comment]
Already fixed in #54. @cdellin's got your back.

**Pull request merged**